pytorch_lightning/accelerators/gpu.py

```diff
@@ -27,6 +27,7 @@ def on_train_start(self):
 
     def on_train_end(self):
         # clean up memory
+        self.model.cpu()
         with torch.cuda.device(self.root_device):
             torch.cuda.empty_cache()
```

Comment:
question: the `with` block is to empty only the cache of the root device? I thought `empty_cache` emptied all

Reply:
did a quick search, more confused than before, but found that soumith suggests it here:
https://github.com/pytorch/pytorch/issues/25752#issuecomment-528866347